client/src/containers/SingleArticle/SingleArticle.js

```diff
@@ -21,9 +21,9 @@ class SingleArticle extends Component {
       })
   }
   showArticleImage() {
-    if (this.state.article.articleImage) {
+    if (this.state.article.articleImage != "null") {
```

Comment:
you should not use the string "null" - you can use the type `null` or `undefined` and the check here can be: `if(!this.state.article.articleImage)`